api/src/main/java/net/countercraft/movecraft/craft/type/CraftType.java

```diff
@@ -391,6 +393,13 @@ public static void registerTypeValidator(Predicate<CraftType> validator, String
         /* Optional properties */
         registerProperty(new RequiredBlockProperty("flyblocks", FLY_BLOCKS, type -> new HashSet<>()));
         registerProperty(new RequiredBlockProperty("detectionblocks", DETECTION_BLOCKS, type -> new HashSet<>()));
+        registerProperty(new MaterialSetProperty("directionDependentMaterials", DIRECTIONAL_DEPENDENT_MATERIALS, type -> {
+            var set = EnumSet.of(Material.LADDER, Material.TORCH, Material.LEVER, Material.GRINDSTONE, Material.LANTERN);
```

Comment:
Redstone and soul torches should be added by default.  I think it may be best to put the entire list into `Tags.java` just so we have our material sets defined together.